### PR TITLE
Remove handwritten ToC

### PR DESCRIPTION
**Pull request by Simran-B**
We have a plugin in the main docs to show a page-toc

Also changed link to main docs to "latest"

### CODE REVIEW

File: docs/Drivers/JS/README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ The official ArangoDB low-level JavaScript client.
 
 **Note:** if you are looking for the ArangoDB JavaScript API in
 [Foxx](https://foxx.arangodb.com) (or the `arangosh` interactive shell) please
-check the
-[ArangoDB documentation](https://docs.arangodb.com/3.1/Manual/Foxx/Modules.html#the-arangodb-module)
+refer to the documentation about the
```
Comment (pluma) [inline docs/Drivers/JS/README.md]: "the documentation about x"?

Is that correct English? I'd have thought that'd be "the documentation of x".

**Comment by pluma** [comment]
lgtm :shipit:

**Comment by sleto-it** [comment]
thanks, merging..

**Comment by arxpoetica** [comment]
You guys pulled the rug out from under me. I rely heavily on that TOC. Are these arangojs pages hosted somewhere other than a GitHub with an automated TOC? What am I missing? Why was it removed?

**Comment by sleto-it** [comment]
@arxpoetica docs.arangodb.com/devel/Drivers

**Comment by Simran-B** [comment]
@arxpoetica The docs will be split into several files / pages soon. We will probably put an overview on the Reference top page.

**Pull request merged**